core/templates/domain/collection/collection-node.model.spec.ts

```diff
@@ -65,10 +70,101 @@ describe('Collection node model', () => {
 
   it('should be able to create a new collection node by exploration ID',
     () => {
-      var collectionNode = CollectionNode.createFromExplorationId(
-        'exp_id0');
+      var collectionNode: CollectionNode =
+        CollectionNode.createFromExplorationId('exp_id0');
       expect(collectionNode.getExplorationId()).toEqual('exp_id0');
       expect(collectionNode.doesExplorationExist()).toBe(false);
     }
   );
+
+  it('should be able to detect if exploration is private',
+    () => {
+      var explorationSummaryBackendObject:
+        LearnerExplorationSummaryBackendDict = {
+          last_updated_msec: 1591296737470.528,
+          community_owned: false,
+          objective: 'Test Objective',
+          id: '44LKoKLlIbGe',
+          num_views: 0,
+          thumbnail_icon_url: '/subjects/Algebra.svg',
+          human_readable_contributors_summary: {},
+          language_code: 'en',
+          thumbnail_bg_color: '#cd672b',
+          created_on_msec: 1591296635736.666,
+          ratings: {
+            1: 0,
+            2: 0,
+            3: 0,
+            4: 0,
+            5: 0
+          },
+          status: 'private',
+          tags: [],
+          activity_type: 'exploration',
+          category: 'Algebra',
+          title: 'exp title'
+        };
+      var collectionNodeBackendObject:
```

Comment:
Use let instead of var (to all changes you added)

Reply:
Done!